ProcessLib/LIE/Common/BranchProperty.h

```diff
@@ -0,0 +1,32 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Eigen>
+
+namespace ProcessLib
+{
+namespace LIE
+{
+struct BranchProperty
+{
+	int node_id;
```

Comment:
These members can be set as const by introducing a constructor.

Reply:
yes it can be const but i don't see a definitive reason for it.